plugins/module_utils/mso.py

```diff
@@ -145,6 +145,14 @@ def mso_expression_spec():
     )
 
 
+def mso_expression_spec_ext_epg():
+    return dict(
+        type=dict(type='str', required=True, aliases=['IP']),
```

Comment:
type should be a choice list with only IP as an option for now. I don't think Aliases is necessary here.